.github/workflows/build.yaml

```
@@ -29,13 +25,15 @@ jobs:
                   password: ${{ secrets.GITHUB_TOKEN }}
 
             - name: Set frontend version
-              run: echo "FRONTEND_VERSION=`ci/version app`" >> $GITHUB_ENV
+              run: |
+                  echo "FRONTEND_VERSION=$(jq '.version' package.json | sed 's/"//g')" >> $GITHUB_ENV
```

Apologies, I forgot about the quotes ^^'

Nitpick: you can use the `-r` jq flag to remove them without needing to use sed 

```suggestion
                  echo "FRONTEND_VERSION=$(jq -r '.version' package.json)" >> $GITHUB_ENV
```

---

No worries, thanks for the tip! I forgot to check if `jq` had an option to do it.